docs/index.md

```diff
@@ -7,82 +7,13 @@
 
 {!.version.md!}
 
-Data validation using Python type annotations.
-
-*pydantic* enforces type hints at runtime, and provides user-friendly errors when data is invalid.
-
-Define how data should be in pure, canonical Python; validate it with *pydantic*.
-
-## Sponsors
-
-Development of *pydantic* is made possible by the following sponsors:
-
-<div class="sponsors">
-  <div>
-    <a rel="sponsored" target="_blank" href="https://www.salesforce.com">
-      <img src="./sponsor_logos/salesforce.png" alt="Salesforce" />
-      <span>Salesforce</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://fastapi.tiangolo.com">
-      <img src="./sponsor_logos/fastapi.png" alt="FastAPI" />
-      <span>FastAPI</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://aws.amazon.com">
-      <img src="./sponsor_logos/aws.png" alt="AWS" />
-      <span>AWS</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://explosion.ai">
-      <img src="./sponsor_logos/explosion_ai.png" alt="Explosion" />
-      <span>Explosion</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://tutorcruncher.com/?utm_source=pydantic&utm_campaign=open_source">
-      <img src="./sponsor_logos/tutorcruncher.png" alt="TutorCruncher" />
-      <span>TutorCruncher</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://www.exoflare.com/open-source/?utm_source=pydantic&utm_campaign=open_source">
-      <img src="./sponsor_logos/exoflare.png" alt="ExoFlare" />
-      <span>ExoFlare</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://home.robusta.dev">
-      <img src="./sponsor_logos/robusta.png" alt="Robusta" />
-      <span>Robusta</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://www.sendcloud.com">
-      <img src="./sponsor_logos/sendcloud.png" alt="SendCloud" />
-      <span>SendCloud</span>
-    </a>
-  </div>
-  <div>
-    <a rel="sponsored" target="_blank" href="https://jina.ai">
-      <img src="./sponsor_logos/jina-ai.png" alt="Jina AI" />
-      <span>Jina AI</span>
-    </a>
-  </div>
-</div>
-
-And many more who kindly sponsor Samuel Colvin on [GitHub Sponsors](https://github.com/sponsors/samuelcolvin#sponsors).
-
-<script>
-  // randomize the order of sponsors
-  const ul = document.querySelector('.sponsors')
-  for (let i = ul.children.length; i >= 0; i--) {
-    ul.appendChild(ul.children[Math.random() * i | 0])
-  }
-</script>
+*pydantic* is the most widely used data validation library for Python.
```

Comment:
I wonder about changing this slightly odd convention of referring to pydantic in lower case italics.

I think "Pydantic" probably makes more sense.

Reply:
I think both uses are fine, but on board with standardizing on Pydantic except in code examples, as relevant